beebop/app.py

```diff
@@ -325,7 +327,7 @@ def run_poppunk_internal(sketches: dict,
 
 
 def setup_db_file_stores(
-    species_args: dict,
+    species_args: Any,
```

Comment:
You've made this change because species_args isn't a dict but an object with attributes? Is it worth making a class for this rather than generic Any?

Reply:
good idea have done